src/terra-dev-site/doc/gettingStarted.a.doc.md

```diff
@@ -31,7 +31,36 @@ This component requires the following peer dependencies be installed in your app
 
 ### Running terra-dev-site
```

Comment:
How does this affect projects extending the terra-dev-site webpack configuration? Will they notice a difference?

Reply:
They shouldn't notice a difference. I should put out some example PR's for terra framework etc.